gbs-control.ino

```diff
@@ -6917,8 +6941,39 @@ void loadDefaultUserOptions() {
 //  //system_phy_set_powerup_option(3); // 0 = default, use init byte; 3 = full calibr. each boot, extra 200ms
 //  system_phy_set_powerup_option(0);
 //}
-
+void ICACHE_RAM_ATTR isr (){
```

Comment:
isrOLED ?

Reply:
Not this is for the Rotatory pot I think

Reply:
This the Interrupt Service Routine for the rotary encoder. I opted for this method in hopes that it wouldn't slow the rest of the program. Typically, I was told that an ISR goes before void setup, but can fix placement if necessary

Reply:
"isrRotaryEncoder" is good then. The naming is just to be instantly in the know what the ISR is for when browsing the code :)

Reply:
edited to reflect this change